indent/scala.vim

```diff
@@ -536,7 +542,7 @@ function! GetScalaIndent()
   if prevline =~ '^\s*\*/'
    \ || prevline =~ '*/\s*$'
     call scala#ConditionalConfirm("18")
-    let ind = ind - 1
+    let ind = ind - 1 - scaladoc_extra_ind
```

Comment:
Thanks for these, will make the changes and push again.

From: Ches Martin [mailto:notifications@github.com]
Sent: 02 August 2016 17:37
To: derekwyatt/vim-scala <vim-scala@noreply.github.com>
Cc: Mark Perryman <Mark.Perryman@metaswitch.com>; Author <author@noreply.github.com>
Subject: Re: [derekwyatt/vim-scala] Configure asterisk alignment in multiline comments to match scaladoc recommendation (#129)


In indent/scala.vim<https://github.com/derekwyatt/vim-scala/pull/129#discussion_r73190760>:

> @@ -536,7 +542,7 @@ function! GetScalaIndent()

>    if prevline =~ '^\s*\*/'

>     \ || prevline =~ '*/\s*$'

>      call scala#ConditionalConfirm("18")

> -    let ind = ind - 1

> +    let ind = ind - 1 - scaladoc_extra_ind

I think these might read better if you just make the variable indicate the complete extra indent of the asterisk, as in:

if get(g:, 'scala_scaladoc_indent', 0)

  let star_indent = 2

else

  let star_indent = 1

end

and then

return ind + star_indent

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/derekwyatt/vim-scala/pull/129/files/6bf6f87baa8c907d14276951f867de1875224937#r73190760>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AP050arMg7LoI9Y5P120jWDKbgm4nkUkks5qb3ISgaJpZM4Iocgs>.